edxlearndot/apps.py

```diff
@@ -0,0 +1,23 @@
+"""
+Django app configuration
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from django.apps import AppConfig
+
+
+log = logging.getLogger(__name__)
```

Comment:
nit: No logging done here anymore -- please remove this and the `import logging` above.

Reply:
Done.